app/models/claim.rb

```diff
@@ -23,10 +23,17 @@ class Claim < ApplicationRecord
   belongs_to :school
   belongs_to :provider
 
-  has_many :mentor_trainings
-  has_many :mentors, through: :mentor_trainings
+  has_many(
+    :mentor_trainings,
+    -> { includes(:mentor).order("mentors.first_name", "mentors.last_name") },
+  )
+  has_many(
```

Comment:
While this is possible, it feels like a code smell to me.

Calling `claim.mentor_trainings.without_hours` seems more intuitive, `[resource].[association].[scope]`. What do you think?